styles.css

```diff
@@ -0,0 +1,76 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: "Poppins", sans-serif;
+    text-decoration: none;
+  }
+  body{
+    width: 100%;
+    height: 718px;
+    background-color: #1C1A19;
+  }
+  header{
+    display: flex;
```

Comment:
- [ ] You are using flex in multiple times. For best practice create d-flex class and use in HTML. Please check the [curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times).

- _This is for example if you use multiple times some classes please change them also._

Reply:
Created a flex class